src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Use `name()` instead of `toString()` for getting the String representation of enum constants. This is to ensure that the method does not get overridden, altering the output. The `name()` method will always return the exact name of the enum constant as declared in the enum definition.

Reply:
Use name() instead of toString() for getting the String representation of enum constants. This is to ensure that the method does not get overridden, altering the output. The name() method will always return the exact name of the enum constant as declared in the enum definition.